### PR TITLE
Fixes #1258

### LINKED ISSUE #1258: Translation updates on hold
Hi, I've paused translation updates due to unresolved parsing issue I commented in https://github.com/commons-app/apps-android-commons/commit/96173e26cf07ef0d3f23cb8bb9ca4b21820eddf5#r27787891

Let's try to find a quick way to solve this.

### PR DESCRIPTION
**Pull request by knight-shade**
## Description

Fixes #1258 

Wraps string stored in strings.xml with CDATA to prevent PHP's SimpleXML return null when string contains other tags.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -79,8 +79,8 @@
   <string name="about_license">The Wikimedia Commons app is an open-source app created and maintained by grantees and volunteers of the Wikimedia community. The Wikimedia Foundation is not involved in the creation, development, or maintenance of the app. </string>
   <string name="trademarked_name" translatable="false">Wikimedia Commons</string>
   <string name="about_improve">Create a new &lt;a href=\"https://github.com/commons-app/apps-android-commons/issues\"&gt;GitHub issue&lt;/a&gt; for bug reports and suggestions.</string>
-  <string name="about_privacy_policy"><u>Privacy policy</u></string>
-  <string name="about_credits"><u>Credits</u></string>
+  <string name="about_privacy_policy"><![CDATA[<u>Privacy policy</u>]]></string>
+  <string name="about_credits">[<![CDATA[<u>Credits</u>]]></string>
```
Comment (nicolas-raoul) [inline app/src/main/res/values/strings.xml]: Is the first `[` really needed?

Comment (knight-shade) [reply app/src/main/res/values/strings.xml]: @nicolas-raoul  Yup, the first [ is part of the syntax.
Source: https://en.wikipedia.org/wiki/CDATA

**Comment by maskaravivek** [comment]
Am not very familiar with how wiki translations work.

@nicolas-raoul @misaochan Can you guys confirm if this will fix the issue. :)

**Comment by misaochan** [comment]
@Nikerabbit has mentioned in #1258 that CDATA does indeed fix the problem. To test this, I think we just need to look at the relevant strings in the app and confirm that our own parsing works and that the strings look OK.

**Pull request merged**